app/gather/api/utils.py

```diff
@@ -0,0 +1,252 @@
+# Copyright (C) 2020 by eHealth Africa : http://www.eHealthAfrica.org
+#
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import logging
+import os
+from requests.exceptions import HTTPError
+from aether.python.utils import request
```

Comment:
This method is also defined in `aether.sdk`, if you are using only this method you can remove this dependency and use `from aether.sdk.utils import request` instead.
https://github.com/eHealthAfrica/aether-django-sdk-library/blob/master/aether/sdk/utils.py#L101-L120